docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,108 @@
+
+We have made it smooth for the community to contribute.
+We started actively reviewing pull requests and guiding issues.
+Things started to really take off with [the release of v1](/blog/material-ui-v1-is-out/) in 2018.
+The frontend community was actively migrating to React and in desperate need to move fast.
+
+Before we realized it, we had grown into the most popular React UI components library.
+We were heavily used and customized to build: customer-facing applications, internal tools, and mobile-hybrid applications.
+
+## A larger focus
+
+In our last survey, the number of developers that complained about improving Material Design implementation was down by [60%](/blog/2020-developer-survey-results/#comparison-with-last-year) compared to the year before.
+At the same time, 400% more developers were struggling to customize the components.
+
+It is based on this context that we started to work on v5 in 2019.
+Our primary focus was to revamp the **customization DX**.
+It had become clear that design and DX were key to unlock the next stage of growth.
+
+The new branding comes to support the v5.0.0 release while also creating space for new initiatives that broaden the company horizon. It's a big deal!
+
+Our ultimate goal is to become the most effective and efficient tool to build UIs while making it accessible to the many.
+
+## Our new brand
+
+Let's dig in.
+
+### A new name
+
+We are breaking the strong association with Material Design.
+We have seen too many people confusing Material-UI with a Google effort or a synonym of Material Design.
+
+We are now called **MUI**. It stands for **M**aterials to build **UI**s
+and is pronounced [/ɛmjuːaɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes ourselves, it's already how many people abbreviate us.
+More importantly, it allows for the release of products not directly coupled to Material Design, such as unstyled/headless version of the components, a brand new second design system as an alternative to MD, and more.
+
+### A new domain name
+
+A new name wouldn't be so official without a new domain.
+So, we have moved home from https://material-ui.com/ to https://mui.com/, a very easy-to-type (and find) 3 letters domain.
+
+### New logos
+
+We've tightened up the company's logo to match the new branding.
+
+It's basically the same, to keep it familiar.
+We're keeping the geometrical shape, to resonate with the _building blocks_ idea of the components.
+We're also reducing the emphasis on the smart 3D perspective, stepping away from the notion of elevation Material Design coined.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="" style="width: 302px; margin-bottom: 16px;" />
+
+We are introducing logos for each of our products:
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/products.png" alt="" style="width: 427px; margin-bottom: 16px;" />
+
```

Comment:
(Makes it look like we're claiming to have designed the Figma logo...)

Reply:
True that. @oliviertassinari [just put together a quick little image](https://www.figma.com/file/4uv2dT18rXJPZBbrbpw61q/Marketing-site?node-id=4598%3A25938) to replace with that one, containing only the logos for Core, X, and Templates. I'm calling Templates and not MUI Store because the logo kinda resonates with it (the T shape) and not with the store as a whole.

Reply:
Ok, but what about the design kits?

Reply:
We haven't done one specifically for it, we're just using Figma's... so it seems to make sense to leave it out of it?

Reply:
> we're just using Figma

I guess for me the point was that we don't have an icon for the design assets. We use Figma, but we have Sketch and Adobe XD too.

Reply:
I'll be resolving this one since it's a bit too late for discussing it 😅